build.mill.scala

```diff
@@ -74,18 +79,27 @@ trait UploadModule extends CrossScalaModule with MillNativeImagePublishModule {
 }
 
 def publishSonatype(tasks: mill.main.Tasks[PublishModule.PublishData]) =
```

Comment:
I think we could get rid of `publishSonatype` altogether, and just run `./mill -i mill.scalalib.SonatypeCentralPublishModule/` from the workflow. This can be fixed later, going to merge not to delay things further.